beacon-chain/rpc/prysm/v1alpha1/debug/state_test.go

```diff
@@ -83,7 +82,7 @@ func TestServer_GetBeaconState(t *testing.T) {
 			Slot: slot + 1,
 		},
 	}
-	state := state.BeaconState(st)
+	state := st
 	// since we are requesting a state at a skipped slot, use the same method as stategen
 	// to advance to the pre-state for the subsequent slot
 	state, err = stategen.ReplayProcessSlots(ctx, state, slot+1)
```

Comment:
```suggestion
	// since we are requesting a state at a skipped slot, use the same method as stategen
	// to advance to the pre-state for the subsequent slot
	state, err := stategen.ReplayProcessSlots(ctx, st, slot+1)
```

Reply:
thanks!